dbcon/mysql/ha_mcs_sysvars.h

```diff
@@ -34,12 +34,19 @@ enum mcs_compression_type_t {
 };
 
 // use_import_for_batchinsert
-enum mcs_use_import_for_batchinsert_t {
+enum class mcs_use_import_for_batchinsert_t {
     OFF = 0,
     ON = 1,
     ALWAYS = 2
 };
 
+// select_handler
+enum class mcs_select_handler_t {
```

Comment:
I would call it select_handler_mode_t to clearly outline what is this enum for.

Reply:
Done.